37_environment_lab/envlab.txt

```diff
@@ -3,7 +3,18 @@
   - rm => rm -i
   - cp => cp -i
 
-2. How can you "install" all defined aliases into the current shell? How can you add them permenantly to all shells?
+
+
+alias ll= "ls -l"
+alias rm= "rm -i"
+alias cp ="cp -i"
+
+2. How can you "install" all defined aliases into the current shell? How can you add them permenantly to all shells?

```

Comment:
After running the above `alias` commands, leaving the shell and re-entering (for example if you close the window and open a new one) all aliases defined will disappear.

In order to define permanent aliases you need to add the above alias lines to a startup file (also called rc file) named `$HOME/.bashrc`, specifically in your case it's `/home/roi/.bashrc`